src/etotal.ado

```diff
@@ -47,10 +49,14 @@ Example(s)
 
     Total of existing variable  
         {bf:. etotal hincp [iw=wgtp]}
+	
+	Total of existing variable, data is svyset  
```

Comment:
This needs to be realigned. Looks like it could be due to mix of tabs and spaces. You can convert all tabs to spaces (and vice-versa) under `Edit/Advanced/`.
<img width="376" alt="helpfile" src="https://user-images.githubusercontent.com/55054385/91466516-3ced2b00-e85d-11ea-91fa-7bd2414bc94d.PNG">